source/dofs/dof_handler.cc

```diff
@@ -39,10 +39,6 @@
 
 DEAL_II_NAMESPACE_OPEN
 
-template <int dim, int spacedim>
-const typename DoFHandler<dim, spacedim>::active_fe_index_type
-  DoFHandler<dim, spacedim>::invalid_active_fe_index;
```

Comment:
You need to keep this. The variable is `static const`, but the compiler still needs to allocate a memory location for it.

Reply:
Ah, and indeed the testers fail for exactly this reason.